.github/workflows/check-models.yml

```diff
@@ -0,0 +1,35 @@
+name: Check models 
+on: push
+
+jobs:
+  check-models:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip
+      - name: Install Scivision
+        run: pip install -e .
+      - name: Install chardet
+        run: pip install chardet 
+      - name: Check model catalog
+        run: scivision-check-models
+      - name: Store artifact
```

Comment:
I have a feeling that this section (Store artifact) is unnecessary and doesn't do anything. I'd be tempted to delete it (given I don't think the file is ever created) and see whether it still runs

Reply:
changes addressed in last commit